src/igvfd/schemas/changelogs/tabular_file.md

```diff
@@ -2,7 +2,7 @@
 
 ### Minor changes since schema version 5
 
-* Add `sequence barcodes` to `content_type`.
+* Add `sequence barcodes` and `barcode to sample mapping` to `content_type`.
```

Comment:
why combine entries, and not list it a separate entry?

Reply:
because it is more concise. There is no ticket number or dates stamp that separates each code change anyways. I can change it back if that's what you prefer.

Reply:
done